packages/cli/test/functional/test_site_convert/test_basic_convert/non_markbind_site/404.md

```diff
@@ -0,0 +1,14 @@
+<frontmatter>
+    title: Page not found
+    layout: 404.md
+</frontmatter>
+
+<body style="font-family: monospace; background-color: #f1f1f1;">
```

Comment:
Seems like this is the original template design, could we standardize and fix all test suites as well as their generated HTML?

Reply:
OMG. I forgot bout dis